app/views/angular_templates/cookies_policy.html.haml

```diff
@@ -69,6 +69,12 @@
       = render_cookie_entry( "_pk_hsr, _pk_cvar, _pk_id and _pk_ses", t( "legal.cookies_policy.cookie_matomo_heatmap_desc" ) )
       = render_cookie_entry( "piwik_ignore, _pk_cvar, _pk_id and _pk_ses", t( "legal.cookies_policy.cookie_matomo_ignore_desc" ) )
 
+  - if Spree::Config.cookies_policy_matomo_section && Spree::Config.matomo_url.present?
+    %p
+      = t 'legal.cookies_policy.statistics_cookies_matomo_optout'
```

Comment:
Since these translations happen in Ruby now, we could use the lazy scoping.

Reply:
yes, I thought about that but I'd have to move translation keys under angular-templates.... that's why I didnt do it.